sql-plugin/src/main/spark320/scala/com/nvidia/spark/rapids/shims/ScanExecShims.scala

```diff
@@ -41,7 +41,19 @@ object ScanExecShims {
         (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.STRUCT + TypeSig.MAP +
           TypeSig.ARRAY + TypeSig.BINARY + TypeSig.DECIMAL_128).nested(),
         TypeSig.all),
-      (fsse, conf, p, r) => new FileSourceScanExecMeta(fsse, conf, p, r)),
+      (fsse, conf, p, r) => {
+        // TODO: HybridScan supports DataSourceV2
+        if (HybridFileSourceScanExecMeta.useHybridScan(conf, fsse)) {
+          // Check if runtimes are satisfied: Spark is not Databricks or CDH; Java version is 1.8;
```

Comment:
Why not databricks or CDH? Is it just that we have not tested with these yet?

Reply:
Yes, because have not tested with CDH and Databricks.

Reply:
We don't have confidence that the Hybrid thing support Databricks spark totally. So, for first version, we consider not to support Databricks.